### DISCUSSION TITLE
Could the Windows 10 emoji picker be restored?

### DISCUSSION THREAD

**Discussion by DavidBerdik** [valinet/ExplorerPatcher#340] [Ideas]
Would it be possible for the emoji picker from Windows 10 to be restored using the patcher? (For anyone who does not know what the emoji picker is, hold down the windows key and press the period key.)

The reason I ask is because the Windows 11 emoji picker was redesigned, and I find it very annoying to work with compared to the Windows 10 one. It was much easier to search for and select an emoji directly from the keyboard when using the Windows 10 one. If it is not possible, that is totally understandable, but if it is possible, could it be considered for implementation?

Thanks for your dedication and hard work on such an amazing tool! Thanks to you, most of the things that I do not like about Windows 11 are gone.

**Comment by BraINstinct0** [comment]
I'm not sure if it's related to either the explorer.exe(or twinui) or Start menu, It could be, but I have no idea.
Anyways, The first step would be finding where the win10 version of emoji picker is left, if it still is there(most win10 features are, just hidden.).

**Reply by DavidBerdik** [reply]
> The first step would be finding where the win10 version of emoji picker is left, if it still is there(most win10 features are, just hidden.).

That's pretty much what I figured too, but I do not know where to even begin with that.
